.gitignore

```diff
@@ -19,3 +19,6 @@ fastlane/FastlaneRunner
 # Bundler
 .bundle
 vendor/bundle
+
+# SourcePackages folder
+SourcePackages
```

Comment:
Please also try to add a new line break here at the end of this file to avoid the red mark display on Git 🙏